eip_operator/src/eip.rs

```diff
@@ -237,7 +237,8 @@ pub mod v2 {
                         },
                     },
                 );
-                eip.meta_mut().resource_version = eip_v1.metadata.resource_version.clone();
+                let resource_version = eip_v1.metadata.resource_version.clone();
```

Comment:
After updating Rust I ran in to a clippy error. This change fixes:

`assigning the result of `Clone::clone()` may be inefficient`